.github/workflows/bahmni_metabase_build_publish.yaml

```diff
@@ -0,0 +1,43 @@
+name: Build and Publish
```

Comment:
Can we add the Image name here, something like "Build and Publish bahmni-metabase"

Reply:
Okay , I have updated it. Thank you.